conf/messages.en

```diff
@@ -262,8 +262,12 @@ checkRegisteredDetails.caption = About the applicant
 checkRegisteredDetails.paragraph.1 = You can provide your own contact details on the next page.
 checkRegisteredDetails.private.paragraph.1 = The business this number is registered to has not consented to their name and address being shared.
 checkRegisteredDetails.checkYourAnswersLabel = Check the name and address for EORI number
-checkRegisteredDetails.error.required = Select yes to confirm your EORI details
-checkRegisteredDetails.warning = These are the details we have on record. They should be the details used to register your Economic Operator Registration and Identification (EORI) number.
+checkRegisteredDetails.error.required.individual = Select yes to confirm your EORI details
+checkRegisteredDetails.error.required.organisation = Select yes to confirm the organisation's EORI details
+checkRegisteredDetails.error.required.agent = Select yes to confirm your EORI details
```

Comment:
Apologies, I couldn't locate the error in the prototype. The error message for agent is same as for an individual. Is that expected?

Reply:
Currently an agent cannot get to this page, we should get a new prototype/content at a later date